core/tests/protractor/profileMenuFlow.js

```diff
@@ -54,7 +54,7 @@ describe('Profile menu flow', function() {
           '.protractor-test-profile-link'));
         await waitFor.elementToBeClickable(
           profileLink, 'Could not click on the profile link');
```

Comment:
Here and below, remove the `await waitFor.elementToBeClickable(..` in places where "action.click" is introduced. This is because `action.click` internally does a "waitFor.elementToBeClickable" before clicking on it.

Reply:
I have made the suggested changes.